mlflow/R/mlflow/R/python.R

```diff
@@ -33,7 +33,11 @@ python_mlflow_bin <- function() {
     return(in_env)
   }
   python_bin_dir <- dirname(python_bin())
-  file.path(python_bin_dir, "mlflow")
+  if (.Platform$OS.type == "windows") {
+    file.path(python_bin_dir, "Scripts", "mlflow")
```

Comment:
Also add this line change in your PR description ?

Reply:
Done.